### ISSUE TITLE
Connecting with an empty password

### ISSUE THREAD

**Issue by andy-g** [formulahendry/vscode-mysql#9]
## Specifying an empty password assumes no password.

When I try add a MySQL connection with an empty password it assumes no password is being used and I get an error:
`Error: ER_ACCESS_DENIED_ERROR: Access denied for user '****:****' (using password: NO)`

If I specify an invalid password (not empty), I get:
`Error: ER_ACCESS_DENIED_ERROR: Access denied for user '****:****' (using password: YES)`

There's no way for me to specify that I want to use a password, but that the password is an empty string.

**Comment by formulahendry** [comment]
Just want to double confirm, you are using a password with an empty string, not none password?

**Comment by andy-g** [comment]
Yes, I'm using a password, but password is an empty string. It's a shared
vagrant box so not so easy for me to manually adjust

**Comment by moreirapontocom** [comment]
That's weird. I'm using Vagrant with empty password as well. Here is working.
